certbot/main.py

```diff
@@ -632,15 +632,25 @@ def _csr_obtain_cert(config, le_client):
         logger.debug(
             "Dry run: skipping saving certificate to %s", config.cert_path)
     else:
-        cert_path, _, cert_fullchain = le_client.save_certificate(
+        le_client.save_certificate(
```

Comment:
ack, I thought that logic happened before here. going back to the old _report_new_cert, then.

how are we giving you the path to cert.pem?